chemprop/args.py

```diff
@@ -181,6 +192,15 @@ def bond_features_size(self) -> int:
     def bond_features_size(self, bond_features_size: int) -> None:
         self._bond_features_size = bond_features_size
 
+    @property
+    def bond_descriptors_size(self) -> int:
+        """The size of the bond descriptors."""
+        return self._bond_descriptors_size
+
+    @bond_descriptors_size.setter
+    def bond_descriptors_size(self, bond_descriptors_size: int) -> None:
+        self._bond_descriptors_size = bond_descriptors_size
```

Comment:
`_bond_descriptors_size` is a private variable defined at line 132.

Reply:
yes but `_bond_descriptors_size` is not _actually_ a private attribute. A client can easily access/modify it just like a normal attribute. You need a double underscore ("dunder", `__`) to actually make it "private" (quotes because nothing is ever truly private in python if you try hard enough)